packages/remote-config/android/src/reactnative/java/io/invertase/firebase/config/ReactNativeFirebaseConfigModule.java

```diff
@@ -17,23 +17,54 @@
  *
  */
 
-import com.facebook.react.bridge.*;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.Callback;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.WritableMap;
 import com.google.firebase.FirebaseApp;
+import com.google.firebase.remoteconfig.ConfigUpdateListenerRegistration;
 import com.google.firebase.remoteconfig.FirebaseRemoteConfigFetchThrottledException;
 import io.invertase.firebase.common.ReactNativeFirebaseModule;
+
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import javax.annotation.Nullable;
 
 public class ReactNativeFirebaseConfigModule extends ReactNativeFirebaseModule {
   private static final String SERVICE_NAME = "Config";
   private final UniversalFirebaseConfigModule module;
 
+  private static HashMap<String, ConfigUpdateListenerRegistration> mConfigUpdateRegistrations = new HashMap<>();
+
   ReactNativeFirebaseConfigModule(ReactApplicationContext reactContext) {
     super(reactContext, SERVICE_NAME);
     module = new UniversalFirebaseConfigModule(reactContext, SERVICE_NAME);
   }
 
+  @Override
+  public void onCatalystInstanceDestroy() {
+    super.onCatalystInstanceDestroy();
+
+    Iterator configRegistrationsIterator = mConfigUpdateRegistrations.entrySet().iterator();
+
+    while (configRegistrationsIterator.hasNext()) {
+      Map.Entry pair = (Map.Entry) configRegistrationsIterator.next();
+      String appName = (String) pair.getKey();
```

Comment:
this isn't used. Can remove.

Reply:
This code is to remove all listeners on destroy, right? This is how auth is doing it

Reply:
where is appName used?

Reply:
If you use android studio, it will tell you when variables aren't used:
<img width="380" alt="Screenshot 2023-05-18 at 11 14 44" src="https://github.com/invertase/react-native-firebase/assets/16018629/73c10efa-c2bd-44ce-b830-6d63cc21e3fc">

Reply:
Ahh sorry didn't see the appName thingy. Removed it :)